### PR TITLE
Delete criterion group UI - us126753

### PR DESCRIPTION
**Pull request by imcdonald-d2l**
Adding delete criterion group UI. [Rally ticket](https://rally1.rallydev.com/#/57732444928d/custom/373260458992?detail=%2Fuserstory%2F519540806352).

delete-action-ui

Back-end: https://github.com/Brightspace/lms/pull/13033

### CODE REVIEW

File: editor/d2l-rubric-criteria-group-editor.js [hunk 1]
```diff
@@ -465,6 +493,49 @@ Polymer({
...
+			setTimeout(function() {
+				announce(this.localize('criterionGroupDeleted', 'name', this._groupName));
+			}.bind(this), 2000);
+
+			this.perfMark(`deleteCriterionGroupEnd-${uuid}`);
+			this.logCriterionGroupDeletedAction(`deleteCriterionGroupStarted-${uuid}`, `deleteCriterionGroupEnd-${uuid}`);
+		}.bind(this)).then(function() {
+			deleteButton.removeAttribute('disabled');
+		}).catch(function(err) {
+			deleteButton.removeAttribute('disabled');
```
Comment (semibran) [inline editor/d2l-rubric-criteria-group-editor.js:528]: could be a good use case for `Promise.finally`

**Review by semibran — APPROVED** [review]

**Review by devpow112 — APPROVED** [review]

**Comment by devpow112** [comment]
Just a reminder that we really should google translate french language stuff.
https://github.com/Brightspace/d2l-rubric#addingupdating-lang-term
Just saves us from issues if translations don't come back in time. I think we've missed a bunch so I'll go through next week and do them all.

**Comment by imcdonald-d2l** [comment]
> Just a reminder that we really should google translate french language stuff.
> https://github.com/Brightspace/d2l-rubric#addingupdating-lang-term
> Just saves us from issues if translations don't come back in time. I think we've missed a bunch so I'll go through next week and do them all. I'd like to see if I can figure out a way to automate this as part of the lang format call.

Thanks for the reminder! I'll add translations for these 👍🏻

File: editor/d2l-rubric-criteria-group-editor.js (hunk 1 above)
Comment (imcdonald-d2l) [reply editor/d2l-rubric-criteria-group-editor.js:528]: Good idea! I'll give that a go 👍🏻

**Review by semibran — APPROVED** [review]

**Pull request merged**